src/uu/pathchk/src/pathchk.rs

```diff
@@ -64,8 +65,9 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     // FIXME: TCS, seems inefficient and overly verbose (?)
     let mut res = true;
     for p in paths.unwrap() {
+        let path_str = p.to_string_lossy();
```

Comment:
This looks incorrect. Not sure how to fix this properly though (operate on bytes?)